cypress/e2e/facility_spec/facility.cy.ts

```diff
@@ -61,6 +66,40 @@ describe("Facility Creation", () => {
     cy.url().should("not.include", "/update");
   });
 
+  it("Configure the existing facility", () => {
+    facilityPage.visitUpdateFacilityPage(facilityUrl);
+    facilityPage.clickManageFacilityDropdown();
+    facilityPage.clickConfigureFacilityOption();
+    facilityPage.fillMiddleWareAddress("dev_middleware.coronasafe.live");
+    facilityPage.clickupdateMiddleWare();
+    facilityPage.verifySuccessNotification("Facility updated successfully");
+  });
+
+  it("Create a resource request", () => {
+    facilityPage.visitUpdateFacilityPage(facilityUrl);
+    facilityPage.clickManageFacilityDropdown();
+    facilityPage.clickResourceRequestOption();
+    facilityPage.fillResourceRequestDetails(
+      "Test User",
+      phone_number,
+      "cypress",
+      "Test title",
+      "10",
+      "Test description"
+    );
+    facilityPage.clickSubmitRequestButton();
+    facilityPage.verifySuccessNotification(
+      "Resource request created successfully"
+    );
+  });
+
+  it("Delete a facility", () => {
+    facilityPage.visitUpdateFacilityPage(facilityUrl);
+    facilityPage.clickManageFacilityDropdown();
+    facilityPage.clickDeleteFacilityOption();
+    facilityPage.confirmDeleteFacility();
+  });
```

Comment:
for deleting a facility, district admin have no access, so verify the error message and the API response

Reply:
Done.